control/control.Kubic.xsl

```diff
@@ -27,7 +27,7 @@
     <xsl:copy>
       <xsl:apply-templates select="node()|@*"/>
       <!-- Make sure this is the openSUSE control file! -->
-      <xsl:copy-of select="document('/usr/lib/skelcd/CD1/control.xml')/*/n:software/n:extra_urls"/>
+      <xsl:copy-of select="document('/CD1/control.xml')/*/n:software/n:extra_urls"/>
```

Comment:
it is actually not so easy due to https://github.com/yast/skelcd-control-openSUSE/blob/master/package/skelcd-control-openSUSE.spec#L93

Is possible to check both locations?

Reply:
Yes, see the update. A minor issue is that `xsltproc` then prints a warning `warning: failed to load external entity "/usr/lib/skelcd/CD1/control.xml"`. But does not fail. Is that OK?